auth/session_test.go

```diff
@@ -56,7 +56,8 @@ var _ = Describe("Session Unit Tests", func() {
 		Expect(s).ToNot(BeNil())
 		Expect(s.getCookie()).To(Equal(cookie))
 		Expect(s.expires).ToNot(BeZero())
-		Expect(s.refreshTime).To(BeTemporally("<", s.expires))
+		// non-persistent cookie expiration set to end of time, can't add time to it
+		Expect(s.refreshTime).To(BeTemporally("==", s.expires))
```

Comment:
Well, as it turned out not really, because we'll overflow on duration and end up _adding time to expiration, not subtracting it. As a side note, `math.MaxInt` was added only in go 1.17

```
fmt.Printf("%q\n", time.Until(time.Now().Add(1<<63-1))*20)
"-20ns"
```

https://play.golang.org/p/QnqfVjyR2MI

Will you be ok if I just set date in two week in future to be done with it?